### PR TITLE
Allow specification of dialertimeout

### PR DESCRIPTION
**Pull request by jangie**
The default value for a connection timeout for the http.Transport is zero; in cases where marathon is behind a proxy or similar device that does not explicitly refuse TCP connections, this means that the current Traefik behavior will be to wait for a connection indefinitely. It would be preferable to set some default that is not indefinite, and allow configuration of same if a different timeout is desired.

### CODE REVIEW

File: docs/toml.md [hunk 1]
```diff
@@ -803,6 +803,13 @@ domain = "marathon.localhost"
 # Optional
 #
 # dcosToken = "xxxxxx"
+
+# Override DialerTimeout
+#
+# Optional
+# Default: 60
+# dialerTimeout = 5
```
Comment (emilevauge) [inline docs/toml.md]: I think it should be `60s` and `5s` as `dialerTimeout` is a `time.Duration`.

Comment (jangie) [inline docs/toml.md]: Running with 1s or "1s" causes a parse error, as duration essentially is an int64.

I'll be adding to the traefik.sample.toml now.

Comment (jangie) [inline docs/toml.md]: @emilevauge ready for another review at your earliest convenience.

Comment (emilevauge) [inline docs/toml.md]: > Running with 1s or "1s" causes a parse error, as duration essentially is an int64.
Indeed, there is an issue with duration parsing in traefik. We should be able to parse `1m` or `10s`. I will fix that later :)

**Comment by emilevauge** [comment]
FYI, I will merge this PR after the merge of `v1.1.0-rc2` into master (this will fix the tests).

**Pull request merged**